developer-guides/rest-api/miscellaneous/directory/README.md

```diff
@@ -0,0 +1,44 @@
+# Info
+
+A method, that searches by users or channels on all users and channels available on server. It supports the [Offset, Count, and Sort Query Parameters](../../offset-and-count-and-sort-info/) along with [Query and Fields Query Parameters](../../query-and-fields-info/).
```

Comment:
Maybe we should add a note here that this method doesn't return the count and all that.